react/client/src/components/Input.tsx

```diff
@@ -62,7 +64,8 @@ const InputArea: React.FC<InputFieldProps> = ({
       disabled={disabled}
       endAdornment={
         <InputAdornment position="end">
-          years old
+          {/* years old */}
```

Comment:
Looks like we can remove this now

Reply:
just pushed an update to this branch ... hope that worked!